command/node_status.go

```diff
@@ -149,9 +149,9 @@ func (c *NodeStatusCommand) Run(args []string) int {
 		// Format the nodes list
 		out := make([]string, len(nodes)+1)
 		if c.list_allocs {
-			out[0] = "ID|DC|Name|Class|Drain|Status|Running Allocs"
+			out[0] = "ID|DC|Name|Class|Build|Drain|Status|Running Allocs"
 		} else {
```

Comment:
Can we keep it for only `-verbose`. Most people aren't running mixed clusters so it will just needlessly expand the width of output